src/components/references/ReferenceSearch.tsx

```diff
@@ -107,7 +107,7 @@ export const ReferenceSearch: React.FC<{
   }, 800)
 
   return (
-    <Container>
+    <Container data-cy={'reference-finder'}>
```

Comment:
I don't there is any benefits to wrapping string with brackets, but that's of no consequence